torch/csrc/jit/codegen/cuda/lower_misaligned_vectorization.cpp

```diff
@@ -0,0 +1,536 @@
+#include <torch/csrc/jit/codegen/cuda/lower_misaligned_vectorization.h>
+
+#include <torch/csrc/jit/codegen/cuda/index_compute.h>
+#include <torch/csrc/jit/codegen/cuda/instrumentation.h>
+#include <torch/csrc/jit/codegen/cuda/ir_iostream.h>
+#include <torch/csrc/jit/codegen/cuda/ir_utils.h>
+#include <torch/csrc/jit/codegen/cuda/kernel_ir.h>
+#include <torch/csrc/jit/codegen/cuda/kernel_ir_builder.h>
+#include <torch/csrc/jit/codegen/cuda/kernel_ir_printer.h>
+#include <torch/csrc/jit/codegen/cuda/lower2device.h>
+#include <torch/csrc/jit/codegen/cuda/lower_utils.h>
+#include <torch/csrc/jit/codegen/cuda/predicate_compute.h>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+namespace cuda {
+
+namespace {
+
+class TORCH_CUDA_CU_API MisalignedVectorizationModifier {
```

Comment:
nit: Is `TORCH_CUDA_CU_API` necessary? Declared in an anonymous namespace, so shouldn't it be exposed?

Reply:
I'll remove it.